frontend/src/components/ProjectServiceAccountRow.vue

```diff
@@ -93,17 +104,19 @@ limitations under the License.
               <v-icon>mdi-pencil</v-icon>
             </v-btn>
           </template>
-          <span>Update Service Account</span>
+          <span>Change service account roles</span>
         </v-tooltip>
       </v-list-item-action>
       <v-list-item-action v-if="canManageServiceAccountMembers" class="ml-1">
         <v-tooltip top>
           <template v-slot:activator="{ on }">
             <v-btn v-on="on" icon color="red" @click.native.stop="onDelete">
-              <v-icon>mdi-delete</v-icon>
+              <v-icon v-if="isServiceAccountFromCurrentNamespace">mdi-delete</v-icon>
+              <v-icon v-else>mdi-close</v-icon>
             </v-btn>
           </template>
-          <span>Delete Service Account</span>
+          <span v-if="isServiceAccountFromCurrentNamespace">Delete Service Account</span>
+          <span v-else>Remove service account from project</span>
```

Comment:
```suggestion
          <span v-else>Remove Service Account from Project</span>
```

Reply:
This is the tooltip for an invited / foreign service account.
<img width="702" alt="Screenshot 2020-10-20 at 17 16 03" src="https://user-images.githubusercontent.com/5526658/96606817-fadde400-12f7-11eb-99cf-0a9cf7212480.png">

I think it would be better if the text would indicate this, like `Remove Foreign Service Account from Project`